web-local/config-megabus.yaml

```diff
@@ -61,20 +61,22 @@ kafka:
 
 megabus:
   applicationId: megabus-local
+  refResolverConsumerGroupName: emodb.megabus.resolver
```

Comment:
Is it conventional to use dots instead  of hyphens? Is there a preference or standard practice from a Kafka perspective?

Reply:
This was @thebearmayor's preference.